src/modules/site-v2/templates/_includes/isotype-database.html

```diff
@@ -1,749 +1,80 @@
-              <td>ECA1074</td>
-              <td>None</td>
-              <td>20200815</td>
-              <td>-34.8934668</td>
-              <td>-56.15637202</td>
-              <td>Urban_garden</td>
-              <td>Montevideo, Uruguay</td>
-              <td>Rotting_nut/pod/seed/fruit</td>
-              <td>U. Koziol, C. Martinez</td>
-              <td>B. Rodriguez</td>
-            </tr>
-            <tr>
-              <th scope="row">46</th>
-              <td><em>C. elegans</em></td>
-              <td>ECA1071</td>
-              <td>ECA1074</td>
-              <td>None</td>
-              <td>20200815</td>
-              <td>-34.8934668</td>
-              <td>-56.15637202</td>
-              <td>Urban_garden</td>
-              <td>Montevideo, Uruguay</td>
-              <td>Rotting_nut/pod/seed/fruit</td>
-              <td>U. Koziol, C. Martinez</td>
-              <td>B. Rodriguez</td>
-            </tr>
-            <tr>
-              <th scope="row">47</th>
-              <td><em>C. elegans</em></td>
-              <td>ECA1072</td>
-              <td>ECA1074</td>
-              <td>None</td>
-              <td>20200815</td>
-              <td>-34.8434</td>
-              <td>-56.1199</td>
-              <td>Urban_garden</td>
-              <td>Montevideo, Uruguay</td>
-              <td>Rotting_nut/pod/seed/fruit</td>
-              <td>U. Koziol, C. Martinez</td>
-              <td>B. Rodriguez</td>
-            </tr>
-            <tr>
-              <th scope="row">48</th>
-              <td><em>C. elegans</em></td>
-              <td>ECA1073</td>
-              <td>ECA1074</td>
-              <td>None</td>
-              <td>20200815</td>
-              <td>-34.8434</td>
-              <td>-56.1199</td>
-              <td>Urban_garden</td>
-              <td>Montevideo, Uruguay</td>
-              <td>Rotting_nut/pod/seed/fruit</td>
-              <td>U. Koziol, C. Martinez</td>
-              <td>B. Rodriguez</td>
-            </tr>
-            <!-- /Sample Data -->
-          </tbody>
-        </table>
-      </div>
-      <!-- /Table Placeholder -->
\ No newline at end of file
+{% block custom_head %}
+<style>
+  #isotype-db_filter {
+    margin: 1rem 0 2rem 0
+  }
+  #isotype-db_length {
+    text-align: left;
+  }
+  .badge {
+    color: black
+  }
+  thead {
+    background-color: #e1eae4;
+  }
+
+</style>
+
+{% endblock %}
+
+<div class="table-responsive my-5 isotypeDatabase">
+  <table class="table table-hover table-bordered table-striped" id="isotype-db">
+    <caption class="visually-hidden">A list of strains sortable by species that includes various data about each
+      individual strain.</caption>
+    <thead class="table-secondary align-middle">
+      <tr>
+        <th scope="col">#</th>
+        <th scope="col">Species</th>
+        <th scope="col">Strain</th>
+        <th scope="col">Isotype</th>
+        <th scope="col">Previous Names</th>
+        <th scope="col">Release</th>
```

Comment:
Need to add a column for "Source Lab" between "Previous Names" and "Release." Erik asked to add this column during the design review.